.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description of change
+
+Please write a summary of your changes and why you made them. Be sure to reference any related issues by adding `fixes # (issue)`.
+
```

Comment:
Should we add a section to link to the relevant issue if any ?

Reply:
I was more thinking of just asking to put a link in the PR since we're not using the `fixes` stuff.